core/workspace_svg.js

```diff
@@ -581,6 +581,15 @@ Blockly.WorkspaceSvg.prototype.getFlyout_ = function() {
   return null;
 };
 
+/**
+ * Getter for the toolbox associated with this workspace, if one exists.
+ * @return {Blockly.Toolbox} The toolbox on this workspace.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.getToolbox = function() {
+  return this.toolbox_;
+};
```

Comment:
Do you plan on using this elsewhere? Curious if it is better to expose a potentially null toolbox that folks have to check for, or just add a different method for doing this.  workspace.addToolboxStyle seems a little specific though too.

Reply:
No specific plans, but I know that getting the toolbox is reasonable and it would be nice to have a getter.  getFlyout does more work to figure out which flyout it is, but is otherwise similar.

Reply:
ah right. We already have get flyout. LGTM